### PR TITLE
Fix indentation and other violations of PEP8

### LINKED ISSUE #1: Fix indentation in tests/test_achievements_by_person.py
Indentation in Python should **always** be with spaces, not tabs, also it should be 4 spaces per indentation level.

@matejkrajcovic also please read [PEP8](https://www.python.org/dev/peps/pep-0008/), which is Python's official style guide. Also all development should be done in separate branches and integrated to `master` via Pull Request interface here on GitHub. I think it is also better if somebody else checks that code, so that bugs not obvious to author are uncovered.

### PR DESCRIPTION
**Pull request by matejkrajcovic**
Changed indentation from tabs to 4 spaces and fixed other PEP8 requirements with two exceptions - max line length in test/__init__.py (line 37) and test/test_achievements_by_person.py (lines 7 and 11) because breaking those lines would IMHO reduce readability.

Fixes #1.

### CODE REVIEW

**Comment by matejkrajcovic** [comment]
to @pepol @mrshu

**Comment by pepol** [comment]
@matejkrajcovic looks good, but I think it would be better to split line 11 in test/test_achievements_by_person.py

@mrshu what's your opinion?

File: test/test_achievements_by_person.py [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-							}, 
-							{
-								'age': 50, 
-								'description': 'Stanford speech'
-							}])
-		response = self.client.get('/achievements/Steve_Jobs/')
-		self.assertItemsEqual(response.json['achievements'], expected['achievements'])
-		self.assertTrue(response.status_code == 200)
+    def test_get_achievements_by_person(self):
+        expected = {'achievements': [{'age': 21,
```
Comment (mrshu) [inline test/test_achievements_by_person.py:6]: @matejkrajcovic wouldn't it be better to just load this from a json file? I know, fast tests and everything, but any addition you make here will just clutter this test which I'd argue is a good reason to put the output in a file which could get loaded at this line into `expected`.

File: test/test_achievements_by_person.py [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
+                                      'description': 'Founded Apple Computer Company with Wozniak'},
+                                     {'age': 50,
+                                      'description': 'Stanford speech'}]}
+        response = self.client.get('/achievements/Steve_Jobs/')
+        self.assertItemsEqual(response.json['achievements'], expected['achievements'])
```
Comment (mrshu) [inline test/test_achievements_by_person.py]: @pepol whatever PEP8 says, I cannot see if it's longer than 80 characters but if it is then sure, please do split it.

**Comment by mrshu** [comment]
@matejkrajcovic looks great in general, let's just have some sort of a discussion on that JSON file thing I commented about.

**Comment by matejkrajcovic** [comment]
@pepol @mrshu I think putting expected values to json file would declutter this file but would clutter project structure. It is also self-documenting this test case. What is your opinion?

**Comment by mrshu** [comment]
@matejkrajcovic What I suggested is quite a common standards with tests in big applications.

What I think you should do is whatever suits you best =)

**Pull request merged**